************* Module pip._vendor.distlib.resources
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:24:0: C0103: Constant name "cache" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:32:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:34:23: W0613: Unused argument 'resource' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:34:33: W0613: Unused argument 'path' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:34:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:66:43: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:71:0: R0205: Class 'ResourceBase' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:72:23: W0621: Redefining name 'finder' from outer scope (line 313) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:71:0: R0903: Too few public methods (0/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:96:8: C0103: Constant name "cache" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:96:8: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:163:12: W0201: Attribute 'path' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:163:12: W0201: Attribute 'path' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:110:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:118:0: R0205: Class 'ResourceFinder' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:133:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:148:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:151:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:166:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:170:42: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:169:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:173:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:177:8: C0103: Argument name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:180:15: R1718: Consider using a set comprehension (consider-using-set-comprehension)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:213:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:220:26: E1101: Module 'zipimport' has no '_zip_directory_cache' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:269:12: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:288:4: E1101: Module 'zipimport' has no 'zipimporter' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_vendor/distlib/resources.py:352:4: W0621: Redefining name 'finder' from outer scope (line 313) (redefined-outer-name)

-----------------------------------
Your code has been rated at 8.33/10

